main_script.py

```diff
@@ -0,0 +1,204 @@
+import sys
+import csv
+from constants import (
+    PERFORMANCE_REWARDS_BUDGET,
+    CONSISTENCY_REWARDS_BUDGET,
+    QUOTE_REWARDS_BUDGET,
+    SOLVER_ADDRESSES,
+)
+from gnosis_scan_api import get_block_range
+from execute_sql_queries import (
+    get_auction_range,
+    compute_solver_rewards,
+    compute_quote_rewards,
+    execute_participation_rewards_helper,
+)
+
+
+def main() -> None:
+    year = int(sys.argv[1])
+    month = int(sys.argv[2])
+    day = int(sys.argv[3])
+    start_block, end_block = get_block_range(year, month, day)
+    print(
+        "\nAccounting period from block "
+        + str(start_block)
+        + " until block "
+        + str(end_block)
+        + "."
+    )
+
+    ## Compute prod and barn auction range
+    (
+        prod_start_auction_str,
+        prod_end_auction_str,
+        barn_start_auction_str,
+        barn_end_auction_str,
+    ) = get_auction_range(str(start_block), str(end_block))
+
+    print(
+        "Production auctions from "
+        + prod_start_auction_str
+        + " until "
+        + prod_end_auction_str
+        + ".\n"
+        + "Barn auctions from "
+        + barn_start_auction_str
+        + " until "
+        + barn_end_auction_str
+        + ".\n"
+    )
+
+    FINAL_CONSISTENCY_REWARDS_BUDGET = CONSISTENCY_REWARDS_BUDGET
```

Comment:
This is not a constant if I read the code correctly.

```suggestion
    final_consistency_rewards_budget = CONSISTENCY_REWARDS_BUDGET
```

Reply:
Oh good catch. Fixed.